docs/quick-start.md

```diff
@@ -0,0 +1,92 @@
+# Quick Start Guide
+
+## Overview
+
+This guide is designed to get you up and running working with the Jetpack Monorepo quickly following recommended and supported guidelines. For more detailed information, including alternate local dev environments, running unit tests, best coding practices, and more, you can the [full Development Environment guide here](docs/development-environment.md#clone-the-repository). 
+
+## Installation
+
+Prior to installation, we recommend using [`Homebrew`](https://brew.sh/) to manage installations and [`nvm`](https://github.com/nvm-sh/nvm/) to manage Node.js versions. If you don't already have those installed, you can do so by copy/pasting each of the following commands and running them in your terminal:
+
+- Homebrew: `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
+- nvm: `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash`
+
+The Jetpack Monorepo requires the following to be installed on your machine:
+
+- Node.js (used for build process and our CLI): `nvm install`
+	- You can see the current latest supported version of Node.js in the monorepo root's [`package.json`](https://github.com/Automattic/jetpack/blob/master/package.json) under `engines`
+	- You can switch versions of node using nvm: `nvm ls` to list installed versions of node, `nvm use [version number]` to switch.
+- PNPM (a Node.js package manager): `npm install -g pnpm`
+- Composer (our PHP package manager): `brew install composer`
+- PHP (the language at the core of the WordPress ecosystem): `brew install php@8.0`
+- Once the pre-requisites are installed, you can clone the GitHub repo:
```

Comment:
Should we clone first so that we can use the repo to grab info like version of php and node?